src/renderers/testing/ReactTestMount.js

```diff
@@ -20,6 +20,10 @@ var getHostComponentFromComposite = require('getHostComponentFromComposite');
 var instantiateReactComponent = require('instantiateReactComponent');
 var invariant = require('invariant');
 
+type TestRendererMockConfig = {
+  getMockRef: (element: ReactElement) => Object,
```

Comment:
Not sure how this would be typed, is there a better way to type an object that looks like a DOM node?